menu/cakes/cakeA.html

```diff
@@ -0,0 +1,409 @@
+          </header>
+          <div class="posts related-posts">
+            <article>
+              <a href="#" class="image"><img src="../../images/cakes/cake_pic04.jpg"
+                  alt="kid's rodeo themed birthday cake" /></a>
+              <h3>Cake B</h3>
+              <p>Aenean ornare velit lacus, ac varius enim lorem ullamcorper dolore. Proin aliquam facilisis ante
+                interdum. Sed nulla amet lorem feugiat tempus aliquam.</p>
+              <ul class="actions">
+                <li><a href="../../orders.html" class="button prefill-button" data-order-type="birthday-kids">Order This</a>
+                </li>
+              </ul>
+            </article>
+            <article>
+              <a href="#" class="image"><img src="../../images/cakes/cake_pic01.jpg" alt="wedding cake" /></a>
+              <h3>Cake C</h3>
+              <p>Aenean ornare velit lacus, ac varius enim lorem ullamcorper dolore. Proin aliquam facilisis ante
+                interdum. Sed nulla amet lorem feugiat tempus aliquam.</p>
+              <ul class="actions">
+                <li><a href="../../orders.html" class="button prefill-button" data-order-type="wedding">Order This</a></li>
+              </ul>
+            </article>
+            <article>
+              <a href="#" class="image"><img src="../../images/cakes/cake_pic07.jpg" alt="marble cake" /></a>
+              <h3>Cake D</h3>
+              <p>Aenean ornare velit lacus, ac varius enim lorem ullamcorper dolore. Proin aliquam facilisis ante
+                interdum. Sed nulla amet lorem feugiat tempus aliquam.</p>
+              <ul class="actions">
+                <li><a href="../../orders.html" class="button prefill-button" data-order-type="cupcakes">Order This</a></li>
+              </ul>
+            </article>
+            <article>
+              <a href="#" class="image"><img src="../../images/cakes/cake_pic10.jpg" alt="soccer cake" /></a>
+              <h3>Cake E</h3>
+              <p>Aenean ornare velit lacus, ac varius enim lorem ullamcorper dolore. Proin aliquam facilisis ante
+                interdum. Sed nulla amet lorem feugiat tempus aliquam.</p>
+              <ul class="actions">
+                <li><a href="../../orders.html" class="button prefill-button" data-order-type="holiday">Order This</a></li>
+              </ul>
+            </article>
+          </div>
+        </section>
+
+      </div>
+    </div>
+
+    <!-- Sidebar -->
+    <div id="sidebar">
+      <div class="inner">
+
+        <!-- Search -->
+        <!-- <section id="search" class="alt">
+									<form method="post" action="#">
+										<input type="text" name="query" id="query" placeholder="Search" />
+									</form>
+								</section> -->
+
+        <!-- Menu -->
+        <nav id="menu">
+          <header class="major">
+            <h2>Menu</h2>
+          </header>
+          <ul>
+            <li><a href="../../index.html" id="navbar-selected">Homepage</a></li>
```

Comment:
remove `id="navbar-selected"` and add it to cake subtab

Reply:
thank you, good catch!